EnglishFilter/sections/specific.txt

```diff
@@ -290,6 +290,7 @@ jizzberry.com##.in.is_desktop
 ||katestube.com^$domain=wankoz.com
 download-video.com##.remove-bg-ads
 ||upcomics.org/k4.gif
+learn-c.org,learn-cpp.org,learn-golang.org,learn-html.org,learn-js.org,learn-perl.org,learn-php.org,learncs.org,learnjavaonline.org,learnpython.org,learnrubyonline.org,learnscala.org,learnshell.org,learnsqlonline.org###google-ad-right
```

Comment:
You blocked useful element `Chapters`. I'll exchange this rule.

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/129632363-444ff183-fc37-4441-9cae-ce22cb863a8d.png)

</details><br/>